docs/help.md

```diff
@@ -3,7 +3,9 @@
 All command line arguments for the `scala-steward` application.
 
 ```
-Usage: scala-steward --workspace <file> --repos-file <file> [--git-author-name <string>] --git-author-email <string> [--git-author-signing-key <string>] --git-ask-pass <file> [--sign-commits] [--vcs-type <vcs-type>] [--vcs-api-host <uri>] --vcs-login <string> [--do-not-fork] [--add-labels] [--ignore-opts-files] [--env-var <name=value>]... [--process-timeout <duration>] [--whitelist <string>]... [--read-only <string>]... [--enable-sandbox | --disable-sandbox] [--max-buffer-size <integer>] [--repo-config <uri>]... [--disable-default-repo-config] [--scalafix-migrations <uri>]... [--disable-default-scalafix-migrations] [--artifact-migrations <uri>]... [--disable-default-artifact-migrations] [--cache-ttl <duration>] [--bitbucket-server-use-default-reviewers] [--gitlab-merge-when-pipeline-succeeds] [--github-app-id <integer> --github-app-key-file <file>] [--url-checker-test-url <uri>] [--default-maven-repo <string>] [--refresh-backoff-period <duration>]
+Usage:
+    scala-steward validate-repo-config
```

Comment:
```suggestion
    scala-steward validate-repo-config path/to/.scala-steward.conf
```

Reply:
I think this file is [auto-generated](https://github.com/scala-steward-org/scala-steward/blob/main/modules/docs/mdoc/help.md) from `decline`'s `Command` instance, which unfortnately does not generate placeholders for the arguments of subcommands. The help mesage can still be obtained via 
```
scala-steward validate-repo-config --help
```

Reply:
I see